src/api/index.ts

```diff
@@ -271,21 +213,14 @@ router.post(
 		const syncType = req.body.syncType || "partial";
 		// Defaults to anything not completed
 		const statusTypes = req.body.statusTypes as string[] || ["FAILED", "PENDING", "ACTIVE"];
+		// Defaults to any installation
+		const installationIds = req.body.installationIds as number[] || [];
 		// Can be limited to a certain amount if needed to not overload system
 		const limit = Number(req.body.limit) || undefined;
 		// Needed for 'pagination'
 		const offset = Number(req.body.offset) || 0;
```

Comment:
Since the function itself has been moved to the model, we might want to add the defaults there too?